homeworks/03-data-structure/src/SimpleBinaryTree.ts

```diff
@@ -0,0 +1,83 @@
+import BinaryTree, { TreeNode, TraverseType } from './BinaryTree'
+
+export default class SimpleBinaryTree<T> implements BinaryTree<T> {
```

Comment:
Why`SimpleBinaryTree`? It's weird... Why not just `BinaryTree`?

If you have some collisions, you can rename imports.

Reply:
Definitely, this is not correct, the name should be correct and independent. You have at least two options:

1. Have the same names and rename when needed:
```ts
import IBinaryTree from './BinaryTree';
// or if it's not a default import
// import { BinaryTree as IBinaryTree } from './BinaryTree';
// by the way, I wouldn't use the default import in this case

export default class BinaryTree<T> implements IBinaryTree<T> { /* ... */ }
```

2. Or have a different name for the interface (`I` prefix is a common approach for that):
```ts
export interface IBinaryTree<T> {
  getRoot(): TreeNode<T>
  setRoot(node: TreeNode<T>): this
  traverse(traverseType: TraverseType): T[]
  getColumn(column: number): T[]
}
```